config/environments/development.rb

```diff
@@ -34,14 +36,22 @@
   config.action_mailer.raise_delivery_errors = false
 
   config.action_mailer.perform_caching = false
-  config.action_mailer.default_url_options = {
```

Comment:
In development mode this just posts to logs anyway, so I think actionmailer config isn't that important. I made a user to make sure it didn't get weird and it looks right:

```
[DEBUG] UserMailer#account_created: processed outbound mail in 388.0ms
[INFO ] Delivered mail 60d758db9b84d_a4117c24369f0@Colins-Air.fios-router.home.mail (361.0ms)
[DEBUG] Date: Sat, 26 Jun 2021 12:42:03 -0400
From: no-reply@dcabortionfund.org
To: ttttttest@example.com
Message-ID: <60d758db9b84d_a4117c24369f0@Colins-Air.fios-router.home.mail>
Subject: Your DARIA account has been created
Mime-Version: 1.0
Content-Type: multipart/alternative;
 boundary="--==_mimepart_60d758db85488_a4117c2436874";
 charset=UTF-8
Content-Transfer-Encoding: 7bit


----==_mimepart_60d758db85488_a4117c2436874
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

Hi Tester --

A DARIA account has been created for you! You can head here to log in: https://www.google.com/?q=cats

We recommend logging in with Google. If you don't log in with Google, you will need to reset your password.

If you experience any difficulties, please reach out to your case management director or volunteer contact.

Thanks for your service!

- Team DCAF

----==_mimepart_60d758db85488_a4117c2436874
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

<p>Hi Tester --</p>
<p>A DARIA account has been created for you! You can head <a href="https://www.google.com/?q=cats">here</a> to log in.</p>
<p>We recommend logging in with Google. If you don't log in with Google, you will need to reset your password.</p>
<p>If you experience any difficulties, please reach out to your case management director or volunteer contact.</p>
<p>Thanks for your service!</p>
<p>- Team DCAF</p>

----==_mimepart_60d758db85488_a4117c2436874--

```